qabelbox/src/main/java/de/qabel/qabelbox/helper/Formater.java

```diff
@@ -21,35 +22,50 @@
     private static final DateFormat dateTimeFormat = DateFormat.getDateTimeInstance(DateFormat.MEDIUM, DateFormat.SHORT);
     private static final DateFormat dateFormat = DateFormat.getDateInstance(DateFormat.MEDIUM);
 
-    public static String formatFileSizeHumanReadable(Context context,long filesize) {
-        String result = "";
+    public static String formatFileSizeHumanReadable(Context context, long filesize) {
+
+        String result;
         DecimalFormat df = new DecimalFormat("#.##");
-        Resources res=context.getResources();
+        Resources res = context.getResources();
         if (filesize < KB) {
-            result = filesize + " "+res.getString(R.string.unit_filesyste_bytes);
+            result = filesize + " " + res.getString(R.string.unit_filesyste_bytes);
         } else if (filesize < MB)
-            result = df.format(new Double(filesize) / KB) + " "+res.getString(R.string.unit_filesyste_kb);
+            result = df.format((double) filesize / KB) + " " + res.getString(R.string.unit_filesyste_kb);
         else if (filesize < GB)
-            result = df.format(new Double(filesize) / MB) + " "+res.getString(R.string.unit_filesyste_mb);
+            result = df.format((double) filesize / MB) + " " + res.getString(R.string.unit_filesyste_mb);
         else {
-            result = df.format(new Double(filesize) / GB) + " "+res.getString(R.string.unit_filesyste_gb);
+            result = df.format((double) filesize / GB) + " " + res.getString(R.string.unit_filesyste_gb);
         }
-        return result.replace(",",".");
+        return result.replace(",", ".");
     }
 
     public static String formatDateShort(Date date) {
+
         return dateFormat.format(date);
     }
 
     public static String formatDateShort(long date) {
+
         return formatDateShort(new Date(date));
     }
 
     public static String formatDateTimeShort(Date date) {
+
         return dateFormat.format(date);
     }
 
     public static String formatDateTimeShort(long date) {
+
         return formatDateTimeShort(new Date(date));
     }
+
+    public static boolean isEMailValid(String email) {
+
+        final String EMAIL_PATTERN =
+                "^[_A-Za-z0-9-\\+]+(\\.[_A-Za-z0-9-]+)*@"
+                        + "[A-Za-z0-9-]+(\\.[A-Za-z0-9]+)*(\\.[A-Za-z]{2,})$";
+
```

Comment:
Please use a less restrictive pattern, for example "^.+@.+$" since the accounting server does yet another check and the real test is just sending an email to that address.

The current regex for example doesn't allow '+' which is very much used to "tag" mail addresses.